pkg/registry/singleton.go

```diff
@@ -52,54 +58,102 @@ func (c *localSingletonConfigurator) Clean() error {
 	return c.infraDriver.CmdAsync(c.DeployHost, fmt.Sprintf(deleteRegistryCommand, "sealer-registry"))
 }
 
+func (c *localSingletonConfigurator) UninstallFrom(hosts []net.IP) error {
+	cmd := fmt.Sprintf("rm -f /root/.docker/config.json && %s", shellcommand.CommandUnSetHostAlias())
```

Comment:
why can RegistryConfigurator remove docker cli config?

if you want to clean the login credentials of Registry from docker, use docker logout.